lib/modules/platform/codecommit/codecommit-client.ts

```diff
@@ -0,0 +1,299 @@
+import {
```

Comment:
I know that you have tested this module in: lib/modules/platform/codecommit/index.spec.ts

but I believe we should add tests for each function separately

Reply:
there's nothing to test, it's just an API, and in `index.spec.ts` i'm not testing them, i'm Mocking them.